app/views/admins/approvals.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= csrf_meta_tags %>
```

Comment:
why do you have to add this? I think this is already in `application.html.erb`

Reply:
hi maam, you're right. This was not needed and is already in application.html.erb. The intention there was that at some point, I was receiving an ActionController::InvalidAuthenticityToken error going to approvals.html.erb and the code was one of the solutions supposedly in[ stock overflow ](https://stackoverflow.com/questions/3364492/actioncontrollerinvalidauthenticitytoken). It happened to work when I bumped into the problem the first time in application.html.erb and I thought this was the solution the second time around. 

Will delete this line, maam. I did see that the local host is working well without the line. Thanks